modules/cnf-about-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_irq_affinity_setting_{context}"]
+= About support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask. These IRQ controllers effectively run on CPU 0.
+
+The following are examples of drivers and hardware that Red Hat are aware lack support for IRQ affinity setting. The list is, by no means, exhaustive:
+
+* Some RAID controller drivers, such as `megaraid_sas`
+* Many non-volatile memory express (NVMe) drivers
+* Some LAN on motherboard (LOM) network controllers
+* The driver uses `managed_irqs`
+
+[NOTE]
+====
+The reason they do not support IRQ affinity setting might be associated with factors such as the type of processor, the IRQ controller, or the circuitry connections in the motherboard..
+====
+
+If the effective affinity of any IRQ is set to an isolated CPU, it might be a sign of some hardware or driver not supporting IRQ affinity setting. To find the effective affinity, log in to the host and run the following:
```

Comment:
```suggestion
If the effective affinity of any IRQ is set to an isolated CPU, it might be a sign of some hardware or driver not supporting IRQ affinity setting. To find the effective affinity, log in to the host and run the following command:
```